bundle-workflow/src/run_integ_test.py

```diff
@@ -104,6 +108,7 @@ def main():
     bundle_manifest = BundleManifest.from_file(args.bundle_manifest)
     build_manifest = BuildManifest.from_file(args.build_manifest)
     test_manifest = TestManifest.from_file(args.test_manifest)
+    test_recorder = TestRecorder(args.test_run_id, "integ-test")
```

Comment:
You might want to define a standard constants which defines each of the test suites. 
We would want all the workflows to use same identifiers.

Reply:
do you mean this? https://github.com/opensearch-project/opensearch-build/pull/467/files#diff-7466053dcf4632ffc67632378636660e4b2a95bec30137fca501ab304e298779R15

Reply:
Yeah that would a part of it. I was getting towards something in common which could be used by all workflows not something which TestRecorder can understand. But this is a good start.